ProcessList.h

```diff
@@ -43,6 +43,7 @@ typedef struct ProcessList_ {
    UsersTable* usersTable;
 
    Hashtable* treeSet;
+   Hashtable* treeSet2;
```

Comment:
Could you use consistent naming (both with a number), but preferably with the name hinting at their actual use?

Reply:
Nothing better then `tmpTreeSet` is comming to my ming :smile: